### DISCUSSION TITLE
希望给中枢网关的虚拟事件增加执行和状态条件

### DISCUSSION THREAD

**Discussion by FDrag0n** [XiaoMi/ha_xiaomi_home#29] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/fa263f9b-2e50-40a5-b592-cdd7be69fed2)
如图，希望增加虚拟事件支持

**Comment by JimmyKmi** [comment, marked as answer]
虚拟事件是通知事件，可以使用通知发送以发送虚拟事件。状态条件无法实现，虚拟事件不存在状态。发送文本格式：["事件"]

**Reply by FDrag0n** [reply]
哦哦，在通知看到了，感谢大佬

**Reply by han7794** [reply]
可以检测到虚拟事件发生，但是无法获取到虚拟事件的值，请问如何解决

**Reply by FDrag0n** [reply]
> 可以检测到虚拟事件发生，但是无法获取到虚拟事件的值，请问如何解决

根据我这里测试的情况，获取值需要从实体事件获取，实体通知只能用来向中枢网关触发虚拟事件。

**Comment by FDrag0n** [comment]
![image](https://github.com/user-attachments/assets/7f5fcad4-e982-4f9c-9ec4-a1f1680ae77e)
测试通过，可以通过实体通知来触发和执行，完美！
